src/platform/plugins/shared/workflows_management/public/features/workflow_list/ui/index.tsx

```diff
@@ -67,17 +69,17 @@ export function WorkflowList({ search, setSearch }: WorkflowListProps) {
   };
 
   const handleRunWorkflow = useCallback(
-    (item: WorkflowListItemDto) => {
+    (id: string, event: Record<string, any>) => {
```

Comment:
noticed two issues:
1. inputs are not rendered in the workflow itself and 
2. if I remove the inputs from the workflow, they still appear in the modal even after page reload
https://github.com/user-attachments/assets/607d963c-a21c-47a9-a691-cf603033ff28